cloud/azure/deploy/config.go

```diff
@@ -34,10 +34,16 @@ type AzureContainerAppsConfig struct {
 	MaxReplicas int `mapstructure:"max-replicas"`
 }
 
+type AzImports struct {
+	// An existing resource group to deploy into
+	ResourceGroup string `mapstructure:"resource-group"`
+}
```

Comment:
Not sure if we should move this to the top level of config as a simple `resource-group` property as well. In some ways it feels less like its being imported for consumption and more of a configuration directive.